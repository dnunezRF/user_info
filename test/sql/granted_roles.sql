\set ECHO 0
BEGIN;
\i sql/user_info.sql
\set ECHO all

-- create empty role
CREATE ROLE foo;

-- should return one row (the role itself)
SELECT count(*) FROM granted_roles('foo');

-- create another role and grant it to foo
CREATE ROLE bar;
GRANT bar TO foo;

-- should return two lines
SELECT count(*) FROM granted_roles('foo');

ROLLBACK;
