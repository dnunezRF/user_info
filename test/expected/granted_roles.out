\set ECHO 0
-- create empty role
CREATE ROLE foo;
-- should return one row (the role itself)
SELECT count(*) FROM granted_roles('foo');
 count 
-------
     1
(1 row)

-- create another role and grant it to foo
CREATE ROLE bar;
GRANT bar TO foo;
-- should return two lines
SELECT count(*) FROM granted_roles('foo');
 count 
-------
     2
(1 row)

ROLLBACK;
